src/igvfd/schemas/changelogs/rodent_donor.md

```diff
@@ -3,6 +3,7 @@
 ### Minor changes since schema version 6
 
 * Add `dbxrefs`.
+* Remove `parents`.
 
```

Comment:
need to bump the schema number (it is new version both for human and for mouse)

Reply:
ok